roles/confluent.kafka_connect/defaults/main.yml

```diff
@@ -22,18 +22,25 @@ kafka_connect_default_internal_replication_factor: "{{ [ groups['kafka_broker']
 
 kafka_connect_group_id: connect-cluster
 
+kafka_connect_metrics:
+  enabled: true
+  consumer_interceptors: []
```

Comment:
same here, it looks like the defaults were stripped

Reply:
The templates are taking care of including the ConfluentMetricsReporter and ConfluentMetricsInterceptor(s)

Reply:
I've made a few changes to clean this up